libqtile/widget/notify.py

```diff
@@ -47,6 +47,7 @@ class Notify(base._TextBox):
         ),
         ("audiofile", None, "Audiofile played during notifications"),
     ]
+    capabilities = {'body', 'actions'}
```

Comment:
Are we really supporting actions? Aren't actions supposed be displayed in the notification with the ability to select them? There can be multiple actions per notification.

If we don't display the action then if you `invoke` it, you're doing it blindly.

Reply:
The first action provided in the actions list is the 'default' action, which is typically invoked by clicking anywhere on a notification and does not necessarily need displaying the message.

Perhaps we can put an option to disable actions, in case a user often uses a program that sends multiple actions and would prefer they can see them all?

Reply:
Yes. I don't have any issues with the approach for the default action but that's really all we're supporting.

We _could_ do it with some extra code but I'm not sure there's a neat way of doing it if we want to keep the text in the bar.